chaingpt-research.md

```diff
@@ -18,6 +18,29 @@ Each company has its own unique path to success. Repeating the actions of anothe
 
 An effective use of ChainGPT's experience is to read the information and think about how to apply their approaches to Ubiquity. Some of the information was collected earlier as I followed their growth. For example, the proposal on the Ubiquity OS plugins launchpad was inspired by the success of their launchpad.
 
+### Plan
+
+Preliminary steps to leverage ChainGPT's expertise
+
+- Alex reviews the collected information about the ChainGPT founder.
+After this, Alex and Sergey determine what experience can be applied to Ubiquity.
+Consider the option of revealing the founder's identity and greater publicity.
+
+- Alex reviews the collected information about ChainGPT's marketing and website.
+After this, Alex and Sergey determine what experience can be applied to Ubiquity.
+Consider options for cooperation with marketing agencies similar to those that supported ChainGPT.
+To think that for such success as ChainGPT, we need to focus on a hype-able product, and not necessarily on serious development.
```

Comment:
Well I think that we spent a fair amount of time developing the fundamentals of our business. Now we can embellish the truth and "sell the dream" which is common in sales and marketing.

Think of something like WeWork. They had a real business (albeit not profitable as far as I recall) but Adam Neumann made the idea of coworking spaces sound like something a lot more exciting than office spaces.